Sources/Fullscreen/FrontpageSearch/Cells/FrontpageSearchImageResultCollectionViewCell.swift

```diff
@@ -176,9 +175,13 @@ final public class FrontpageSearchImageResultCollectionViewCell: UICollectionVie
             return
         }
         contentStackView.insertArrangedSubview(remoteImageView, at: 0)
+        if item.imageContentModeFill == false {
+            remoteImageView.contentMode = .scaleAspectFit
+        }
         remoteImageView.isHidden = false
         remoteImageView.dataSource = remoteImageViewDataSource
         remoteImageView.loadImage(for: imageUrl, imageWidth: imageAndButtonWidth)
+        remoteImageView.layer.backgroundColor = .milk
```

Comment:
I see the image now is white also in dark mode. Is it supposed to be like that, due to transparent company logos? 😄

Reply:
It is, but it doesn't work due to how RemoteImageView is set, so I will remove the line as it is misleading to the reader.